.github/workflows/core_build_publish.yml

```diff
@@ -91,26 +91,33 @@ jobs:
           path: ${{ needs.prepare.outputs.artifacts_name }}
   publish:
     runs-on: ubuntu-18.04
-    name: Publish to NuGet
+    name: Publish to feeds
     needs: [prepare, build]
-    if: ${{ needs.prepare.outputs.publish_nuget == '1' }}
+    if: ${{ needs.prepare.outputs.publish_artifacts == '1' }}
     steps:
       - name: Fetch build
         id: download
         uses: actions/download-artifact@v2
         with:
           name: ${{ needs.prepare.outputs.artifacts_name }}
+      - name: Push to Feedz
+        run: |
+          unzip "$ZIP"
+          find -name '*.nupkg' -exec dotnet nuget push -k "$FEEDZ_API_KEY" -s 'https://f.feedz.io/leancode/public/nuget/index.json' -n true '{}' ';'
       - name: Create release
+        if: ${{ needs.prepare.outputs.publish_nuget == '1' }}
         uses: actions/create-release@v1
         with:
           tag_name: ${{ format('v{0}', needs.prepare.outputs.version) }}
           release_name: ${{ format('Release v{0}', needs.prepare.outputs.version) }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Push to NuGet
+        if: ${{ needs.prepare.outputs.publish_nuget == '1' }}
         run: |
           unzip "$ZIP"
           find -name '*.nupkg' -exec dotnet nuget push -k "$NUGET_API_KEY" -s 'https://api.nuget.org/v3/index.json' -n true '{}' ';'
         env:
           NUGET_API_KEY: ${{ secrets.NUGET_API_KEY }}
+          FEEDZ_API_KEY: ${{ secrets.FEEDZ_API_KEY }}
```

Comment:
This `env` block is scoped to the Publish to NuGet step, so publishing to nuget will fail. I think it's possible to setup env on job level to avoid copying, or if not this would have to go to the correct step